docs/setup_docs.sh

```diff
@@ -36,14 +36,25 @@ echo "Created temporary file" $goModFileLocation/go.mod
 currentBranch=$(git rev-parse --abbrev-ref HEAD)
 
 function integrate_connector_docs {
+  local connector ref additional_folders
   connector=$1
   ref=$2
+  additional_folders=( "${@:3}" )
+
   git clone --single-branch --branch ${ref} https://github.com/apache/flink-connector-${connector}
   theme_dir="../themes/connectors"
   mkdir -p "${theme_dir}"
-  rsync -a flink-connector-${connector}/docs/content* "${theme_dir}/"
+
+  for rsync_folder_subpath in "docs/content*" "${additional_folders[@]}"; do
```

Comment:
```suggestion
  for rsync_folder_subpath in "docs/content" "docs/content.zh" "${additional_folders[@]}"; do
```
sorry; my bad. The if statement doesn't allow the asterix. AFAIU, the asterix was just used initially to rsync both `docs/content` and `docs/content.zh`.

Reply:
This was also sloppy on my part, should have tried it locally first 😅